src/annotator/highlighter/overlay-highlighter/overlay-highlight.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const overlayUtils = require('./overlay-utils');
+
+class OverlayHighlight {
+
+  constructor(normedRange, target = document.body){
+    this.normedRange = normedRange;
+    this.range = normedRange.toRange();
+    this.target = target;
+
+    const highlightContainer = overlayUtils.createSVGElement('svg');
+    const highlightEl = overlayUtils.createSVGElement('g');
+
+    this.container = highlightContainer;
+    this.container.style.position = 'absolute';
+
+    // Disable pointer events
```

Comment:
This comment tells me what the code does, but it would be helpful to change it to something that says, briefly, why.